node/cli/src/command.rs

```diff
@@ -877,7 +878,7 @@ impl CliConfiguration<Self> for RelayChainCli {
 	fn base_path(&self) -> Result<Option<BasePath>> {
 		Ok(self
 			.shared_params()
-			.base_path()
+			.base_path()?
```

Comment:
Did it change the return type?

Reply:
Yes, sort of, in substrate `a684e831b2a12e696240168a3fcc9c6affcf146b` (#6221):

```diff
-   fn base_path(&self) -> Result<Option<PathBuf>> {
+   fn base_path(&self) -> Result<Option<BasePath>> {
```

We may want to revisit what the following line is doing as a result of this...